config_examples/template_config.yaml

```diff
@@ -54,32 +55,37 @@ telescope:
 defaults:
     # This must be a string specifying an absolute path to a default
     # configuration file or one of the season default keywords
-    default_config: 'h2c'
+    'h2c'
 systematics:
     rfi:
         # see hera_sim.rfi documentation for details on parameter names
         rfi_stations:
+            seed: once
             stations: !!null
         rfi_impulse:
-            chance: 0.001
-            strength: 20.0
+            impulse_chance: 0.001
```

Comment:
Ah, so this is because of modifications I made to the `rfi` module to keep some of the `defaults` logic simple. Basically, `defaults` unravels whatever configuration it's given so that any level of dictionary nesting is removed--so having multiple instances of `chance` or `std` (rather than differentiating things as, e.g., `scatter_chance` and `impulse_chance`) would break `defaults` handling of `rfi` component parameters.

Reply:
On the face of it this seems a bit brittle -- what if there are other modules in the future that also require this kind of handling? But I'm sure you have good reason for it, and there might not be a "better" solution.

Reply:
It's been a while since I wrote the `defaults` module, but I believe that the motivation here was to keep the code relatively simple and to make the API for setting new defaults as simple as possible. To be honest, I'm not really sure what the *right* thing to do is... implementing package-wide, dynamic defaults handling is hard, so it's entirely possible that I haven't converged on the "correct" solution.

Reply:
That sounds like the right goal haha. Let's go with this, and hopefully if issues arise we can tweak as we go.